### PR TITLE
Create Campaign Flow - Initial non-published state #1583

### PR DESCRIPTION
**Pull request by HamidMosalla**
Closes: #1583

### CODE REVIEW

**Comment by mgmccarthy** [comment]
@HamidMosalla, I'm going to have a look at this today at some point.

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Campaigns/PublishViewModelQuery.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AllReady.Areas.Admin.ViewModels.Campaign;
```
Comment (mgmccarthy) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Campaigns/PublishViewModelQuery.cs]: @HamidMosalla, please remove these un-needed usings from the class

**Comment by HamidMosalla** [comment]
@mgmccarthy Thanks! looking forward to it.

**Comment by HamidMosalla** [comment]
@mgmccarthy Thanks for the review, I committed the requested changes.

**Comment by mgmccarthy** [comment]
@tonysurma :shipit:

**Comment by tonysurma** [comment]
Thank you @HamidMosalla

**Pull request merged**